lpeg/io_modules/decoders/logfmt.lua

```diff
@@ -0,0 +1,135 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+# Logfmt Message Decoder Module
+
+## Decoder Configuration Table for input cfg file
+decoder_module = "decoders.logfmt"
+
+-- Allows you to remap parsed logfmt tags into another field
+logfmt_conf = {
+    -- override root fields using Fields fields
+    remap_hash_root = {
```

Comment:
I would prefer not to encourage fixing-up/altering schemas in real-time (as opposed to fixing it at the source, I also understand that fixing the source is not always possible).  However, we have been actively removing mandatory restrictions/hard limits, in Hindsight, and simply providing recommendations to allow this flexibility when needed... so lets keep it but label it with a warning/caveats.  Also, you say you are adding similar functionality in multiple places?  Can it be factored out and put in 'transformation' module?  Or at least share a common naming conventions/structure?